packages/amplication-notification-service/tsconfig.app.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["node"],
+    "emitDecoratorMetadata": true,
+    "target": "es2015"
```

Since it is a new project, let's try to start with the right foot. 🦶 

```suggestion
    "target": "es2022",
    "strict": true,
```